packages/eip-5792-middleware/src/hooks/processSendCalls.test.ts

```diff
@@ -432,5 +437,199 @@ describe('EIP-5792', () => {
         `EIP-7702 upgrade not supported as account type is unknown`,
       );
     });
+
+    it('validates auxiliary funds with unsupported account type', async () => {
+      await expect(
+        processSendCalls(
+          sendCallsHooks,
+          messenger,
+          {
+            ...SEND_CALLS_MOCK,
+            from: FROM_MOCK_HARDWARE,
+            capabilities: {
+              auxiliaryFunds: {
+                optional: false,
+                requiredAssets: [
+                  {
+                    address: '0x123',
+                    amount: '0x2',
+                    standard: 'erc20',
+                  },
+                  {
+                    address: '0x123',
+                    amount: '0x2',
+                    standard: 'erc20',
+                  },
+                ],
+              },
+            },
+          },
+          REQUEST_MOCK,
+        ),
+      ).rejects.toThrow(
+        'Unsupported non-optional capabilities: auxiliaryFunds',
+      );
+    });
+
+    it('validates auxiliary funds with unsupported chain', async () => {
+      isAuxiliaryFundsSupportedMock.mockReturnValue(false);
+
+      await expect(
+        processSendCalls(
+          sendCallsHooks,
+          messenger,
+          {
+            ...SEND_CALLS_MOCK,
+            capabilities: {
+              auxiliaryFunds: {
+                optional: false,
+                requiredAssets: [
+                  {
+                    address: '0x123' as Hex,
+                    amount: '0x1' as Hex,
+                    standard: 'erc20',
+                  },
+                ],
+              },
+            },
+          },
+          REQUEST_MOCK,
+        ),
+      ).rejects.toThrow(
+        `The wallet no longer supports auxiliary funds on the requested chain: ${CHAIN_ID_MOCK}`,
```

Comment:
🤔 I know this error is not added in this PR, but I'm just seeing the language and wondering if it should be changed to
`The wallet does not support auxiliary funds on the requested chain: `